src/core/agent/services/identifierService.ts

```diff
@@ -36,6 +35,8 @@ class IdentifierService extends AgentService {
     "Failed to rotate AID, operation not completing...";
   static readonly FAILED_TO_OBTAIN_KEY_MANAGER =
     "Failed to obtain key manager for given AID";
+  static readonly IDENTIFIER_ALREADY_EXISTS =
+    "The identifier has already been incepted";
```

Comment:
`IDENTIFIER_NAME_TAKEN = "Identifier name has already been used on KERIA";` - and also put `${IdentifierService.IDENTIFIER_NAME_TAKEN}: ${name}`